main.cpp

```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
         c) pass the local variables to ignoreUnused() as you did in 2b)
     see main() for an example of this.
  
-10) click the [run] button.  Clear up any errors or warnings as best you can.
- */
+10) click the [run] buttn.  Clear up any errors or warnings as best you can.
 
+*/
 
 //2)
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+    int myHeight = 72;    
+    int howManyTrees = 8;
+    int moneyEarned = 0;
+
+    float temperature = 32.0f; 
+    float rotations = 2.375f;
+    float exactChange = 12.86f;
+
+    bool doorIsOpen = true;
+    bool makeASound = false;
+    bool activateSwitch = true;
+
+    double numberOfTracks = 4; 
+    double inputsAllowed = 24;
+    double eqModesShown = 6;
+
+    unsigned int balloonsPopped = 350;
+    unsigned int numberOfPlanesSeen = 10;
+    unsigned int horsesCompeting = 24;
+    
+    ignoreUnused(number, myHeight, howManyTrees, moneyEarned, temperature, rotations, exactChange, doorIsOpen, makeASound, activateSwitch, numberOfTracks, inputsAllowed, eqModesShown, balloonsPopped, numberOfPlanesSeen, horsesCompeting); //Passing each variable declared to the ignoreUnused() function
 }
 
 /*
- 10 functions
- example:
- note: this example shows the result after completing steps 3-8
- */
-bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
-    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
-/*
- 1)
- */
+Declare 10 free functions
+note: this example shows the result after completing steps 3-8
+*/
 
-/*
- 2)
- */
-
-/*
- 3)
- */
-
-/*
- 4)
- */
-
-/*
- 5)
- */
-
-/*
- 6)
- */
-
-/*
- 7)
- */
-
-/*
- 8)
- */
-
-/*
- 9)
- */
-
-/*
- 10)
- */
+bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused function
+    return {}; //if your function returns anything other than 'void,' add 'return {}};' to the end of it 
+}
+
+//1)
+bool lowPassFilter(int slopeType = 2, float cutoffFreq = 100.0f)
+{
+    ignoreUnused(slopeType, cutoffFreq);
+    return {};
+}
+
+//2)
+int distanceWalkedToday(int numbOfSteps, float milesPerHour = 12.3f)
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`

- [ ] revise